packages/wonder-blocks-announcer/src/util/dom.ts

```diff
@@ -0,0 +1,82 @@
+import {
+    type PolitenessLevel,
+    RegionDictionary,
+} from "../../types/Announcer.types";
+
+/**
+ * Create a wrapper element to group regions for a given level
+ * @param {string} level Politeness level for grouping
+ * @returns {HTMLElement} Wrapper DOM element reference
+ */
+export function createRegionWrapper(level: PolitenessLevel) {
+    const wrapper = document.createElement("div");
+    wrapper.id = `wbAWrap-${level}`;
+    return wrapper;
+}
+
+/**
+ * Create multiple live regions for a given level
+ * @param {HTMLElement} wrapper Parent DOM element reference to append into
+ * @param {string} level Politeness level for grouping
+ * @param {number} regionCount Number of regions to create
+ * @param {RegionDictionary} dictionary Reference to Announcer dictionary
+ * @returns {HTMLElement[]} Array of region elements
+ */
+export function createDuplicateRegions(
+    wrapper: HTMLElement,
+    level: PolitenessLevel,
+    regionCount: number,
+    dictionary: RegionDictionary,
+): HTMLElement[] {
+    const result = new Array(regionCount).fill(0).map((el, i) => {
+        const region = createRegion(level, i, dictionary);
+        wrapper.appendChild(region);
+        return region;
+    });
+    return result;
```

Comment:
**question:** Is the return value only needed for tests? I see that it is not used in the code, so I'm wondering if that return would be useful in the `Announcer` class in some way?